server.js

```diff
@@ -3,39 +3,46 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const cors = require ('cors');
 const mongoose = require('mongoose');
+
 //Instantiate app
 const app = express();
-app.use(bodyParser.json());
-app.use(cors());
-//Require mongoDB models
-const user = require('./models/user');
-const exercise = require('./models/exercise');
+
 //Connect to the database
 mongoose.connect(process.env.MONGODB_URI  || 'mongodb://localhost/exercise');
+
+//Require mongoDB models
+const User = require('./models/user');
+const Exercise = require('./models/exercise');
+
+// include middleware
+app.use(bodyParser.json());
+app.use(cors());
```

Comment:
You won't need this cors module if the client and server will be on the same port.